plugin/src/main/scala/org/flywaydb/play/ConfigReader.scala

```diff
@@ -32,49 +35,65 @@ class ConfigReader(configuration: Configuration, environment: Environment) {
   def getFlywayConfigurations: Map[String, FlywayConfiguration] = {
     (for {
       dbName <- getAllDatabaseNames
+      // Will only iterate if this is a configuration group, instead of a single value, allowing easy skipping of irrelevant entries
+      subConfig <- catching(classOf[PlayException]).opt(configuration.getConfig("db." + dbName)).flatten
+
+      // Skip silently if we've been told to
+      if !subConfig.getBoolean("flywayIgnore").getOrElse(false)
```

Comment:
Is this option really needed?
You can ignore a database by putting no migration scripts for it.